### PR TITLE
replace cachito npm-without-deps with hermeto copy

### PR DESCRIPTION
**Pull request by derasdf**
merge all npm-without-deps PR's after approval at the same time
AFTER WHICH WILL REQUIRE HERMETO-SIDE TEST UPDATE

Also chagne cachi2-yarn-berry commit from cachito-testing to yarn/zero-installs from hermetoproject/integration-tests in readme

tested in hermeto with
```
HERMETO_TEST_INTEGRATION_TESTS_REPO=https://github.com/derasdf/integration-tests.git \
HERMETO_GENERATE_TEST_DATA=true \
python -m pytest \
    'tests/integration/test_yarn.py::test_e2e_yarn[yarn_e2e]' \
    -v

### CODE REVIEW

**Comment by eskultety** [comment]
I had an issue running hermeto ITs as
```
HERMETO_TEST_INTEGRATION_TESTS_REPO=https://github.com/derasdf/integration-tests.git HERMETO_GENERATE_TEST_DATA=1 
python -m pytest  tests/integration/test_yarn.py::test_e2e_yarn[yarn_e2e] -v
```

**Comment by eskultety** [comment]
> I had an issue running hermeto ITs as
> 
> ```
> HERMETO_TEST_INTEGRATION_TESTS_REPO=https://github.com/derasdf/integration-tests.git HERMETO_GENERATE_TEST_DATA=1 
> python -m pytest  tests/integration/test_yarn.py::test_e2e_yarn[yarn_e2e] -v
> ```

My bad, the variable is now called `HERMETO_TEST_GENERATE_DATA`, working.

**Review by eskultety — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
     "yargs": "^17.7.2"
   },
   "resolutions": {
-    "ccto-wo-deps": "patch:ccto-wo-deps@https://github.com/cachito-testing/cachito-npm-without-deps/archive/2f0ce1d7b1f8b35572d919428b965285a69583f6.tar.gz#./.yarn/patches/ccto-wo-deps-git@github.com-e0fce8c89c.patch"
+    "ccto-wo-deps": "patch:ccto-wo-deps@https://github.com/hermetoproject/integration-tests/archive/b7dacdbf112ce0ea17a75c389e9e0f2f70f5be81.tar.gz#./.yarn/patches/ccto-wo-deps-git@github.com-e0fce8c89c.patch"
```
[the remark this answers is not in the document]
Comment (eskultety) [reply package.json:33]: tests passing -> resolved.
